src/sentry/static/sentry/app/components/customResolutionModal.tsx

```diff
@@ -59,7 +72,7 @@ export default class CustomResolutionModal extends React.Component {
               onChange={this.onChange}
               placeholder={t('e.g. 1.0.4')}
               url={url}
-              onResults={results => {
+              onResults={(results: Release[]) => {
```

Comment:
wouldn't it be better to move it to a separate function? Usually I try to avoid inline functions, because it can cause performance issues.

Reply:
true true 👍